src/spikeinterface/sorters/launcher.py

```diff
@@ -181,7 +196,13 @@ def run_sorter_jobs(job_list, engine="loop", engine_kwargs={}, return_output=Fal
                 f.write(slurm_script)
                 os.fchmod(f.fileno(), mode=stat.S_IRWXU)
 
-            subprocess.Popen(["sbatch", str(script_name.absolute()), f"-cpus-per-task={cpus_per_task}", f"-mem={mem}"])
+            progr = [sbatch_executable]
+            for k, v in engine_kwargs.items():
+                progr.append(f"--{k}")
+                progr.append(f"{v}")
+            progr.append(str(script_name.absolute()))
+            p = subprocess.run(progr, capture_output=True, text=True)
```

Comment:
will `subprocess.run()` mean this is no longer run asynchronously vs. `subprocess.Popen()`?

Reply:
Sbatch returns immediately after submitting the job to the queue, so it's still going to be asynchronous. The run() syntax is just a simpler way to interact with the subprocess module and the one I have more experience with, that's all

Reply:
Okay nice ofc sbatch will just return, agree `run` is the preferred way to use `subprocess`. Nice!